### ISSUE TITLE
about the 54version

### ISSUE THREAD

**Issue by ArsNova123** [HelloRicky123/Siamese-RPN#10]
I noticed that the config.py in your 54version is different from the former version ,the instance_size of 54ver is 271 while the instance_size of the former ver is 255.Does this mean that data preprocessing needs to be done again when running version 054 of the code?

**Comment by HelloRicky123** [comment]
no

**Comment by ArsNova123** [comment]
> no

OK,thank you,I followed your settings to train this model and found the loss turned into nan,I slightly reduced the initial learning rate.It can run properly.

**Comment by Holdeny** [comment]
> > no
> 
> OK,thank you,I followed your settings to train this model and found the loss turned into nan,I slightly reduced the initial learning rate.It can run properly.

Could you tell me the initial learning rate you set? I have the same problem as you, thank you.

**Comment by HelloRicky123** [comment]
All setting is in the 59version folder without any warmup.------------------ Original ------------------From: Holdeny  <notifications@github.com>Date: Sat,Mar 30,2019 11:35 PMTo: HelloRicky123/Siamese-RPN <Siamese-RPN@noreply.github.com>Cc: HelloRicky123 <qftju12345@163.com>, Comment <comment@noreply.github.com>Subject: Re: [HelloRicky123/Siamese-RPN] about the 54version (#10)

no

OK,thank you,I followed your settings to train this model and found the loss turned into nan,I slightly reduced the initial learning rate.It can run properly.

Could you tell me the initial learning rate you set? I have the same problem as you, thank you.

—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or mute the thread.
{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/HelloRicky123/Siamese-RPN","title":"HelloRicky123/Siamese-RPN","subtitle":"GitHub repository","main_image_url":"https://github.githubassets.com/images/email/message_cards/header.png","avatar_image_url":"https://github.githubassets.com/images/email/message_cards/avatar.png","action":{"name":"Open in GitHub","url":"https://github.com/HelloRicky123/Siamese-RPN"}},"updates":{"snippets":[{"icon":"PERSON","message":"@Holdeny in #10: \u003e \u003e no\r\n\u003e \r\n\u003e OK,thank you,I followed your settings to train this model and found the loss turned into nan,I slightly reduced the initial learning rate.It can run properly.\r\n\r\nCould you tell me the initial learning rate you set? I have the same problem as you, thank you."}],"action":{"name":"View Issue","url":"https://github.com/HelloRicky123/Siamese-RPN/issues/10#issuecomment-478256814"}}}
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/HelloRicky123/Siamese-RPN/issues/10#issuecomment-478256814",
"url": "https://github.com/HelloRicky123/Siamese-RPN/issues/10#issuecomment-478256814",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by HelloRicky123** [comment]
Closed
